UI/window-basic-main.hpp

```diff
@@ -71,6 +71,8 @@ class OBSBasicStats;
 #define SIMPLE_ENCODER_NVENC "nvenc"
 #define SIMPLE_ENCODER_AMD "amd"
 
+#define SIMPLE_ENCODER_NVENC_HEVC "nvenc_hevc"
```

Comment:
Move this between SIMPLE_ENCODER_NVENC, and SIMPLE_ENCODER_AMD.

Reply:
Done.